src/browser/workspace-api.ts

```diff
@@ -103,7 +103,12 @@ export class RestDevWorkspaceApi implements IDevWorkspaceApi {
 
   async delete(namespace: string, name: string): Promise<void> {
     await this.axios.delete(
-      `/apis/${devWorkspaceApiGroup}/${devworkspaceVersion}/namespaces/${namespace}/${devworkspacePluralSubresource}/${name}`
+      `/apis/${devWorkspaceApiGroup}/${devworkspaceVersion}/namespaces/${namespace}/${devworkspacePluralSubresource}/${name}`, {
+        headers: {
+          'content-type': 'application/json; charset=utf-8',
+        },
+        data: deletionOptions(deletePolicy.Background)
```

Comment:
I wonder if some client may need to set Foreground, in case they don't use websockets for tracking when removal is finished.
Then we can add deletion options optional as parameters, with background as default.
But it may make more sense to do such change when we have at least one client that will need it.